basilisk/src/PointTriangle.h

```
@@ -0,0 +1,314 @@
+        *s = 1.0;
+        d2 = a00 + 2.0*b0 + c;
+      }
+      else
+      {
+        *s = -b0/a00;
+        d2 = b0**s + c;
+      }
+    }
+    else  // region 0
+    {
+      if (det == 0.) // degenerate triangle
```

Clang (14.0.6)

[note] +3, including nesting penalty of 2, nesting level increased to 3

(clang)